extensions/wikia/ArticleVideo/bower_components/html5-skin/scripts/jest/environment.js

```diff
@@ -0,0 +1 @@
+__DEV__ = true;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "let", "const" or "var" keyword to this declaration of "__DEV__" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2703)